apps/daimo-mobile/src/view/TabNav.tsx

```diff
@@ -344,6 +345,10 @@ function DepositTab() {
         name="LandlineTransfer"
         component={LandlineTransferScreen}
       />
+      <DepositStack.Screen
+        name="BitrefillWebView"
```

Comment:
same as above -- can this just be `Bitrefill` or `BitrefillOfframp` or something other than webview